src/storage/disk_manager.c

```diff
@@ -3073,7 +3073,7 @@ disk_verify_volume_header (THREAD_ENTRY * thread_p, PAGE_PTR pgptr)
   DISK_VOLUME_HEADER *vhdr;
 
   assert (pgptr != NULL);
-  (void) pgbuf_check_page_ptype (thread_p, pgptr, PAGE_VOLHEADER);
+  /* (void) pgbuf_check_page_ptype (thread_p, pgptr, PAGE_VOLHEADER); */
```

Comment:
This line is commented out instead of making it run only in debug mode like other code. Can I know the reason?

Reply:
I'll fix it. thanks.

Reply:
Well, it's been a while so I don't remember exactly.. I think I was trying to check the performance change in debug mode.